### PR TITLE
translate 2013-09-09-network-link-conditioner.md and 2014-06-16-clang-di...

### PR DESCRIPTION
**Pull request by aprilnov**
...agnostics.md

### CODE REVIEW

File: 2013-09-09-network-link-conditioner.md [hunk 1]
```diff
@@ -0,0 +1,111 @@
+---
+title: Network Link Conditioner
+author: Mattt Thompson
+category: Xcode
+tag: popular
+excerpt: "Product design is about empathy. Knowing what a user wants, what they like, what they dislike, what causes them frustration, and learning to understand and embody those motivations in design decisions—this is what it takes to make something insanely great."
+translator: April Peng
+excerpt: "产品设计是一种同情。知道用户想要什么，他们喜欢什么，他们不喜欢什么，是什么原因会让他们沮丧，学习去理解并且把那些动机决实现于设计 - 这就是把一些疯狂的事情做得漂亮要去做的事情。"
```
Comment (croath) [inline 2013-09-09-network-link-conditioner.md]: empathy：同感心、感同身受......

File: 2013-09-09-network-link-conditioner.md [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+---
+
+Product design is about empathy. Knowing what a user wants, what they like, what they dislike, what causes them frustration, and learning to understand and embody those motivations in design decisions—this is what it takes to make something insanely great.
+
+产品设计是一种同情。知道用户想要什么，他们喜欢什么，他们不喜欢什么，什么原因会让他们沮丧，学习去理解并且把那些动机决实现于设计 - 这就是把一些疯狂的事情做得漂亮需要去做的事情。
```
Comment (croath) [inline 2013-09-09-network-link-conditioner.md]: empathy...

**Comment by croath** [comment]
LGTM

**Comment by croath** [comment]
@rickytan Any other ideas?

**Comment by rickytan** [comment]
明天再来看看， hold on

File: 2013-09-09-network-link-conditioner.md [hunk 3]
```diff
@@ -0,0 +1,69 @@
+---
+title: Network Link Conditioner
+author: Mattt Thompson
+category: Xcode
+tag: popular
+translator: April Peng
+excerpt: "产品设计是一种感同身受。知道用户想要什么，他们喜欢什么，他们不喜欢什么，是什么原因会让他们沮丧，学习去理解并且把那些动机决实现于设计 - 这就是把一些疯狂的事情做得漂亮要去做的事情。"
+---
+
+产品设计是一种感同身受。知道用户想要什么，他们喜欢什么，他们不喜欢什么，什么原因会让他们沮丧，学习去理解并且把那些动机决实现于设计 - 这就是把一些疯狂的事情做得漂亮需要去做的事情。
```
Comment (rickytan) [inline 2013-09-09-network-link-conditioner.md]: 决实现于设计，什么意思？还有，用中文破折号吧

**Comment by rickytan** [comment]
done

**Comment by aprilnov** [comment]
改了~ 没问题就 merge 吧~ 阿里嘎多~

**Comment by natecook1000** [comment]
Published! :tada:

Awesome work—you're cranking them out so fast! :laughing:

**Pull request merged**